src/reference/security/backup.md

```diff
@@ -1,4 +1,4 @@
-% last review: 2023-10-26
+% last review: 2024-10-26
```

Comment:
is this scheduled or done? same for security

Reply:
I've reviewed and found nothing which had to be changed.